recipe/meta.yaml

```diff
@@ -37,17 +36,12 @@ requirements:
     - click >=0.5
 
 test:
-  requires:
-    - pytest
-    - hypothesis
   imports:
     - skan
-  commands:
-    - py.test --pyargs skan --doctest-modules
```

Comment:
The problem is definitely not `skan` and the package should work just fine in an environment with X11. I'm removing the full tests here so you can get this merged and upload the new version. (I'm working on the mpl side too to see if I can fix this.)

Reply:
I've seen some packages test by starting a xvfb service before running things, maybe that works?

Reply:
Yep, [it can be done]( https://github.com/conda-forge/qtconsole-feedstock/blob/49232460fb21032b328c0c3c8e57a7b057772ded/recipe/meta.yaml#L31-L33 ). Though usually it's a bit tricky getting this to work in all cases. Depends how important that is for us to test here. 🙂

Reply:
I met @ocefpaf in the hallway and he persuaded me that it's not that important. =P Merging!

Reply:
... Though of course as soon as I did that I thought, maybe I should just ignore that one troublesome module... =) Oh well, this is something for *after* SciPy!

Reply:
Thank you everyone for the help!!!